contracts/tests/Spigot.t.sol

```diff
@@ -232,6 +232,21 @@ contract SpigotTest is Test {
         assertSpigotSplits(address(token), totalRevenue);
     }
 
+    function test_claimRevenue_pushPaymentToken_bad_revenue_contract(uint256 totalRevenue) public {
```

Comment:
can u make test name a but more discriptive like "cantClaimRevenueOnNonexistantContract"

Reply:
*ContractWithNoSettings" might be better